resources/styles/components/task-step/reading/book-content/work-in-physics.less

```diff
@@ -28,4 +28,16 @@
 
   }
 
+  figure {
+    width: 100%;
+    margin: 0;
+    img {
+      width: 100%;
+    }
+  }
+
+  div[data-type="title"] + figure + p {
```

Comment:
I'm hesitant to modify the selector in `book-content/index.less` since that apply to all IReadings.

I took a stab at changing it to also use `first-of-type` but ran into problems with it matching `p` elements in the objectives, and not matching the page.   It would also match the first para on any reading, even ones that are a continuation of a previous page.  That may be desired behavior though.

Since it's been working fine thus far, I'd suggest we leave it until we can make a guided review of what elements it should and shouldn't match.